AutomatticTracks/src/main/java/com/automattic/android/tracks/crashlogging/CrashLogging.kt

```diff
@@ -1,7 +1,43 @@
 package com.automattic.android.tracks.crashlogging
 
 interface CrashLogging {
-    fun log(throwable: Throwable)
-    fun log(throwable: Throwable, data: Map<String, String?>)
-    fun log(message: String)
+
+    /**
+     * Records a breadcrumb during the app lifecycle but doesn't report an event. This basically
+     * adds more context for the next reports created and sent by [sendReport] or an unhandled
+     * exception report.
+     *
+     * @param[message] The message to attach to a breadcrumb
+     * @param[category] An optional category
+     */
+    fun recordEvent(
+        message: String,
+        category: String? = null
+    )
+
+    /**
+     * Records a breadcrumb with exception during the app lifecycle but doesn't report an event.
+     * This basically adds more context for the next reports created and sent by [sendReport] or an
+     * unhandled exception report.
+     *
+     * @param[exception] The message to attach to a breadcrumb
+     * @param[category] An optional category
+     */
+    fun recordException(
+        exception: Throwable,
+        category: String? = null
+    )
```

Comment:
`breadcrumbs` for an exception looks like this:
<img width="954" alt="image" src="https://user-images.githubusercontent.com/5845095/114701873-a99d3300-9d23-11eb-8569-48f6600d593c.png">